.travis.mk

```diff
@@ -282,6 +282,7 @@ test_static_build_cmake_linux: deps_tests
 
```

Comment:
Please, add a prefix into the commit title. Look at git log for an appropriate prefix.

https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message

Reply:
I don't see such prefix in https://github.com/tarantool/tarantool/commit/07b103832f0002c78cb77b715d861b7d876f7f45